Mathlib/Tactic/ToAdditive.lean

```diff
@@ -618,13 +618,13 @@ def warnExt [Inhabited σ] (stx : Syntax) (ext : PersistentEnvExtension α β σ
     (thisAttr attrName src tgt : Name) : CoreM Unit := do
   if f (ext.getState (← getEnv)) src then
     Linter.logLintIf linter.existingAttributeWarning stx <|
-      m!"The source declaration {src} was given attribute {attrName} before calling @[{thisAttr}]. {
-      ""}The preferred method is to use `@[{thisAttr} (attr := {attrName})]` to apply the {
-      ""}attribute to both {src} and the target declaration {tgt}." ++
+      m!"The source declaration {src} was given attribute {attrName} before calling @[{thisAttr}]. \
```

Comment:
suggestion doesn't work, but consider hanging the `m!"\` here instead of `<|` on the previous line

Reply:
I was thinking of hanging `m!"\`, but I thought that it would make it less clear that there's the `++ if` logic later. I think I'd rather leave this as a direct translation and then let Floris deal with this the next time he touches this part of ToAdditive.